core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -421,6 +421,7 @@ object KafkaConfig {
   val ZkSyncTimeMsDoc = "How far a ZK follower can be behind a ZK leader"
   val ZkEnableSecureAclsDoc = "Set client to use secure ACLs"
   val ZkMaxInFlightRequestsDoc = "The maximum number of unacknowledged requests the client will send to Zookeeper before blocking."
+
```

Comment:
Changes in this file are not needed.

Reply:
yeah, I'll replace with the file from trunk in next push